apps/cyberstorm-nextjs/utils/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const API_DOMAIN =
+  process.env.NEXT_PUBLIC_API_DOMAIN || "https://thunderstore.io";
```

Comment:
Should this app have a `.env.development` file for such env variables if we're using them?

Reply:
Yes, I'll add it 💯